charts/rancher-turtles/0.3.0+up0.11.0/values.yaml

```diff
@@ -0,0 +1,86 @@
+rancherTurtles:
+  image: registry.rancher.com/rancher/rancher/turtles
+  imageVersion: v0.11.0
+  imagePullPolicy: IfNotPresent
+  namespace: rancher-turtles-system
+  managerArguments: []
+  imagePullSecrets: []
+  features:
+    cluster-api-operator:
+      cleanup: true
+      kubectlImage: rancher/kubectl:v1.30.3
+    embedded-capi:
+      disabled: true
+    rancher-webhook:
+      cleanup: true
+      kubectlImage: rancher/kubectl:v1.30.3
+    rancher-kubeconfigs:
+      label: false
+    managementv3-cluster:
+      enabled: true
+    managementv3-cluster-migration:
+      enabled: false
+    propagate-labels:
+      enabled: false
+    etcd-snapshot-restore:
+      enabled: false
+    addon-provider-fleet:
+      enabled: false
+cluster-api-operator:
+  enabled: true
+  cert-manager:
+    enabled: false
+  volumes:
+    - name: cert
+      secret:
+        defaultMode: 420
+        secretName: capi-operator-webhook-service-cert
+    - name: clusterctl-config
+      configMap:
+        name: clusterctl-config
+  volumeMounts:
+    manager:
+      - mountPath: /tmp/k8s-webhook-server/serving-certs
+        name: cert
+        readOnly: true
+      - mountPath: /config
+        name: clusterctl-config
+        readOnly: true
+  cluster-api:
+    enabled: true
+    configSecret:
+      name: ""
+      defaultName: capi-env-variables
+    core:
+      namespace: capi-system
+      imageUrl: "registry.opensuse.org/isv/suse/edge/clusterapi/containers/images/cluster-api-controller:1.7.5"
+      fetchConfig:
+        url: ""
+        selector: ""
+    rke2:
+      enabled: true
+      version: ""
+      bootstrap:
+        namespace: rke2-bootstrap-system
+        imageUrl: "registry.opensuse.org/isv/suse/edge/clusterapi/containers/images/cluster-api-provider-rke2-bootstrap:0.6.1"
```

Comment:
Shouldn't we bump `cluster-api-provider-rke2-bootstrap` to `v0.7.0`?

Reply:
Updated, thanks!